docs/readmes/architecture/agw_overview.md

```diff
@@ -0,0 +1,59 @@
+---
+id: agw_overview
+title: AGW Overview
+hide_title: true
+---
+
+# Access Gateway Overview
+
+The main service within the access gateway (AGW) is Magmad, which brings up all the services hosted within the AGW. The other major services and components hosted within the AGW are: the MME, S-PGW, Health Checker, Mobilityd, Sessiond/PCEF, Pipelined, PolicyDB, Subscriberdb, OVS-data path, Enodebd, and the Control Proxy. Together, these components help to facilitate and manage data both to and from the user. 
+
+![AGW diagram](assets/agw_services.png)
+
+## Access Gateway Services and  Components
+
+1. **Sctpd** - High availability SCTP interface to radio front end. Sctpd forms the endpoint of the S1-C connection. 
```

Comment:
High availability? I think we should simply state that `Termination service for the SCTP connections currently used for S1-C and N2 interfaces`

Reply:
Will update - thanks!